README.adoc

```diff
@@ -23,6 +23,10 @@ We created *Smart Testing* to give you the fastest possible feedback loop when i
 
 ifndef::generated-doc[]
 To read complete documentation head over to http://arquillian.org/smart-testing/
+
+You can also generate it locally. All you need to have in place is [docker](https://docs.docker.com/engine/installation/).
+Simply execute `$ ./.asciidoctor/generate.sh` and you are set.
```

Comment:
Add copy to clipboard extension?

Reply:
It will not work, as this part is only rendered on GH.

Reply:
ah true :)